lib/claim.js

```diff
@@ -51,14 +51,21 @@ const requestVariables = () => [
 ];
 
 const validate = (vars) => {
+  const isNotProduction = (process.env.NODE_ENV === 'staging' || process.env.NODE_ENV === 'development');
```

Comment:
I think this is probably better represented by a comparison with `production` 
```suggestion
  const isNotProduction = process.env.NODE_ENV !== 'production';
```